token/program-2022/src/extension/mod.rs

```diff
@@ -907,12 +926,15 @@ pub enum ExtensionType {
     MetadataPointer,
     /// Mint contains token-metadata
     TokenMetadata,
-    /// Test variable-length mint extension
     /// Mint contains a pointer to another account (or the same account) that holds group
     /// configurations
     GroupPointer,
+    /// Test variable-length mint extension
     #[cfg(test)]
-    VariableLenMintTest = u16::MAX - 2,
+    VariableLenMintTest = u16::MAX - 4,
+    /// Test fixed-length mint extension
+    #[cfg(test)]
+    FixedLenMintTest = u16::MAX - 3,
```

Comment:
What does this give you over `MintPaddingTest`?

Reply:
Just felt like it made more sense to have specific test extensions for specific tests, but if I can use `MintPaddingTest` in the fixed-length tests I can do that!